packages/profile-sync-controller/package.json

```diff
@@ -41,12 +41,23 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@metamask/base-controller": "^5.0.2",
+    "@metamask/keyring-api": "^6.1.1",
+    "@metamask/notifications-controller": "^1.0.0",
+    "@metamask/snaps-controllers": "^8.1.1",
+    "@metamask/snaps-sdk": "^4.2.0",
+    "@metamask/snaps-utils": "^7.4.0",
     "@noble/ciphers": "^0.5.2",
     "@noble/hashes": "^1.4.0",
+    "deepmerge": "^4.2.2",
```

Comment:
This dependency is already present in `devDependencies`.

Reply:
Fixed.